Include AddTutorialCommandTest

suwoons opened the pull request:
Implementing Attendance sub-feature of Tutorial in progress

suwoons committed:
Update UG

suwoons committed:
Merge https://github.com/AY1920S2-CS2103-W15-3/main

suwoons committed:
AddTutorialCommand test

suwoons committed:
Merge https://github.com/AY1920S2-CS2103-W15-3/main

hanchingyong commented on docs/UserGuide.adoc:
```diff
@@ -263,16 +263,16 @@ Format: `markPresent INDEX week/WEEK student/STUDENT [OPTIONAL]`
-* `markPresent 2 week/7 student/John Tan` +
-Marks only John Tan as present in week 7 for tutorial at index 2
-* `markPresent 2 tutorialName/T02 week/7` +
+* `markPresent 2 week/7 student/2` +
```
tutorial at index 2 or 3?

suwoons committed:
Implement AddTutorialStudentCommand (w/o storage)

In midst of implementing AddTutorialStudentCommand. Have not updated JSON storage for Tutorial to include enrolled students or the attendance sheet yet.
Also updated UG and the GUI icon.

suwoons committed:
Fix checkstyle and tests

suwoons commented:
Haven't implemented JSON storage for enrolledStudents and attendance in Tutorial yet; in progress!!

suwoons committed:
Add tutorial exceptions

CranstonYeo commented:
lgtm
